scripts/sts/create-rosa-sts.sh

```diff
@@ -0,0 +1,260 @@
+#!/bin/bash
```

Comment:
This script is old so should be removed in favour of https://github.com/integr8ly/delorean/pull/254

Reply:
I will remove it in my next PR